recipe/meta.yaml

```diff
@@ -20,12 +20,11 @@ requirements:
     - python >=3.7
   run:
     - addict
-    - matplotlib-base
     - numpy
-    - opencv >=3
```

Comment:
Dropping these heavy `matplotlib` and `opencv` dependencies by using `MMENGINE_LITE=1 pip install .` following https://github.com/open-mmlab/mmengine/pull/1362. Users can still install these dependencies manually if needed.

Hopefully this doesn't affect `mmcv` and other downstream dependencies like mmdet/mmseg. Note that the `mmcv` conda-forge package intentionally skips adding `opencv` as mentioned at https://github.com/conda-forge/mmcv-recipe-feedstock/blob/61fb2e9f7ebb4e5abbb0986033cd96cf3312dd45/recipe/meta.yaml#L48-L52.